spinalcordtoolbox/download.py

```diff
@@ -46,8 +46,9 @@
     },
     "PAM50": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20230519/PAM50-r20230519.zip",
-            "https://osf.io/8cgme/?action=download",
+            # DO NOT MERGE. This is a prerelease. This should be superseded by a new PAM50 link once spinal level changes are merged.
+            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20231011/PAM50-r20231011.zip",
+            "https://osf.io/dj93u/?action=download",
```

Comment:
It looks like adding the label `60` caused one of our existing tests to fail:

```
test_sct_label_vertebrae_high_value_warning - AssertionError: assert 'Disc value not included in template.' in 'WARNING  
```
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/78d46e0d1a75dfa6a27689931bf4111a233c04f1/testing/cli/test_cli_sct_label_vertebrae.py#L70-L75

What _should_ happen is that `-initz 40,19` (z=40, label=19) should trigger this warning:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/78d46e0d1a75dfa6a27689931bf4111a233c04f1/spinalcordtoolbox/vertebrae/core.py#L208-L212

(This happens because we start with `19`, then increment in the inferior direction (`20`), and then `list_disc_value_template.index(current_value)` fails with `current_value==20`)

I will try to debug why the addition of the `60` label causes this to _not_ happen.

Reply:
Issue opened here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4254

Reply:
Resolved by https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4250/commits/de1019ee67d94e829160a28210894ed3636afd1f and https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4250/commits/0187e613cfddaa2bf7f1cc2ae32c11988d7bc8ee.

For justifications for these changes, see https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4254#issuecomment-1758581360.

Reply:
These changes have been migrated to https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4268.